doc/source/cluster/usage-stats.rst

```diff
@@ -0,0 +1,52 @@
+.. _ref-usage-stats
+
+Usage Stats Collection
+======================
+
+Starting Ray 1.13, Ray collects usage stats data by default and the data will be used by the Ray team to understand how to improve Ray.
+For more context, please refer to this `RFC <https://github.com/ray-project/ray/issues/20857>`_.
+
+
+How does it work
+----------------
+When a Ray cluster is started via ``ray start --head``, ``ray up``, ``ray submit --start`` or ``ray exec --start``,
+Ray will decide whether usage stats collection should be enabled or not by considering the following factors in order:
+
+#. Check whether environment variable ``RAY_USAGE_STATS_ENABLED`` is set: 1 means enable and 0 means disable.
+
+#. If the environment variable is not set, then read the value of key ``usage_stats`` in the global config file ``~/.ray/config.json``: true means enable and false means disable.
+
+#. If neither is set and the user is running the ray command interactively, then the user will be prompted to enable or disable the collection. If the command is run non-interactively, usage stats collection will be enabled by default. The decision will be saved to ``~/.ray/config.json`` so the prompt is only shown once.
+
+Note: usage stats collection is not enabled for local dev cluster started via ``ray.init()``.
+
+If usage stats collection is enabled, a background process on the head node will collect the usage stats
+and report to ``https://usage-stats.ray.io/`` every hour. The reported usage stats will also be saved to
+``/tmp/ray/session_xxx/usage_stats.json`` on the head node for inspection. You can check the existence of this file to see if collection is enabled.
+
+Usage stats collection is very lightweight and should have no impact on your workload in any way.
+
+What data is collected
```

Comment:
Why don't we put this on the top and "how does it work" should be at the bottom.

Reply:
My assumption is users care things in this order
1. what data is collected
2. How to disable it
3. How it works

Reply:
I feel `How it works` makes understanding `How to disable it` easier.

Reply:
I agree with @rkooo567's comment. We could minimally add a link to the  `How to disable it?` header in the introductory paragraph, similar to:

> Starting in Ray 1.13, Ray will collect API utilization statistics by default (guarded by an opt-out prompt, [see below]()). These data will be used by the open-source Ray engineering team to better understand how to improve our libraries and core APIs, and how to prioritize bug fixes and enhancements. For more context, please refer to this `RFC <https://github.com/ray-project/ray/issues/20857>`_.

cc: @richardliaw @ericl